testdata/test-filesystem/gopath/src/tags/tagged.go

```diff
@@ -0,0 +1,7 @@
+// +build godebugtest
+
+package main
+
+func init() {
+	v = "y"
+}
```

Comment:
Thanks for adding test files. I don't see test cases that use them, though. Can you add those?